packages/eslint-plugin/src/rules/unified-signatures.ts

```diff
@@ -616,7 +616,15 @@ function getOverloadInfo(node: OverloadNode): string {
     default: {
       const { key } = node as MethodDefinition;
 
-      return isIdentifier(key) ? key.name : (key as TSESTree.Literal).raw;
+      if (isPrivateIdentifier(key)) {
+        return `privateId_${key.name}`;
```

Comment:
[Nit]: The returned string mixes camel-case and underscore-case. What do you think about making it consistent (e.g. `private_identifier_${key.name}`)?

I'd personally also use the whole word (`identifier`) instead of its shorter name (`id`), but that's just my personal preference, feel free to push me back on this 👍

This is also relevant for the other string returned below.

Reply:
You're right, it's better to unify and clarify the naming. I've made the corrections with 15f802f.